gddo-server/assets/site.js

```diff
@@ -119,6 +119,40 @@ $(function() {
 
 });
 
+// highlighting function
+$(function() {
+    var clickedElem = "";
+
+    function escapeSelector(s) {
+        return s.replace(/(:|\.|\[|\])/g, "\\$1");
+    }
+
+    var highlightElement;
+    highlightElement = function (el) {
+        $(escapeSelector(clickedElem)).removeClass("highlighted");
+        clickedElem = $(el).attr('href');
+        $(escapeSelector(clickedElem)).addClass("highlighted");
+    };
+
+    if ("onhashchange" in window) {
+        var highlightedSel = "";
+        window.onhashchange = function() {
+            if (highlightedSel) {
+                $(highlightedSel).removeClass("highlighted");
+            }
+            highlightedSel = window.location.hash.replace( /(:|\.|\[|\]|,)/g, "\\$1" );
+            if (highlightedSel) {
+                $(highlightedSel).addClass("highlighted");
+            }
+        }
+        window.onhashchange();
+    }
+
+    $(".indexEntry", ".permalink").click(function() {
+        highlightElement(this);
```

Comment:
why are you calling the highlightElement function instead of just putting the body of the function here?